### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by atgeirr**
This fixes a compile issue for me, and silences warnings.

### CODE REVIEW

**Comment by atgeirr** [comment]
jenkins build this please

File: tests/parser/ScheduleTests.cpp [hunk 1]
```diff
@@ -1338,6 +1338,7 @@ DATES             -- 1
 )";
 
     const auto& schedule = make_schedule(input);
+    static_cast<void>(schedule); // Silence unused variable warning.
```
Comment (joakim-hove) [inline tests/parser/ScheduleTests.cpp:1341]: Just remove the entire test - it is not interesting. In the case the test actually makes sense to have I usually implement it with `BOOST_CHECK_NO_THROW()`:
```C++
BOOTS_CHECK_NO_THROW( make_schedule( input ) );
```

Comment (bska) [inline tests/parser/ScheduleTests.cpp:1341]: Any reason to not just disable this entire check completely?  It doesn't actually do anything as far as I can tell.  I'd probably just put `#if 0` ahead of `BOOST_AUTO_TEST_CASE` and `#endif` after the tests's closing brace.

**Comment by atgeirr** [comment]
I'll remove the test.

**Comment by atgeirr** [comment]
jenkins build this please

**Pull request merged**